sig-ci/README.md

```diff
@@ -8,7 +8,8 @@ This SIG is focused on CI (continuous integration) experience within Spinnaker.
 ## Meetings
 
 * Regular SIG Meeting
-  * Every other Tuesday, 1 PM EST / 10AM PST
+
```

Comment:
Did the Tuesday meeting ever make it to the [spinnaker google calendar](https://calendar.google.com/calendar/embed?src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc%40group.calendar.google.com&ctz=America%2FLos_Angeles)?  Trying to find out if I've missed any meetings.

Reply:
Yes - the same Tuesdays as the spinnaker as code sig (but morning).

Reply:
OK, I think I figured it out...I wasn't looking far enough into the future.  The first one I see is currently Wednesday Feb 12th at 10AM PST.

Reply:
BTW, noticed https://github.com/spinnaker/governance/pull/96 along the way.